profile/README.md

```diff
@@ -1,2 +1,12 @@
-The Concepticon project presents an attempt to link the large amount of different concept lists which are used in the linguistic literature, 
+The Concepticon project is an attempt to link the large amount of different concept lists which are used in the linguistic literature, 
 ranging from Swadesh lists in historical linguistics to naming tests in clinical studies and psycholinguistics.
+
+The resulting Concepticon (in the sense of [Poornima and Good 2010](https://aclanthology.org/W10-2101.pdf)) facilitates
+- comparison of lexical data across languages and collection contexts and
+- cross-linguistic comparison of data associated with word forms or concepts collected in the NoRaRe sub-project.
```

Comment:
In other texts, we used "words and concepts" instead of "word forms", so I would exchange "word forms" with "words" to make it consistent.

Reply:
done